### DISCUSSION TITLE
Where should I put the static data for version 14.2.14?

### DISCUSSION THREAD

**Discussion by Alan0818zz** [vercel/next.js#70827] [Help]
### Summary

Where should I put the static data for version 14.2.14?

**Comment by Juneezee** [comment]
You can put your static assets in the `public` folder. See https://nextjs.org/docs/app/building-your-application/optimizing/static-assets

**Reply by Alan0818zz** [reply]
I can't find the public folder for version 14.14.2.14.

**Reply by Alan0818zz** [reply]
![螢幕擷取畫面 2024-10-05 235057](https://github.com/user-attachments/assets/49e4a6c1-f380-4cb4-ae36-e1201386fb34)

**Reply by Juneezee** [reply]
You can create a `public` folder at the same level as the `app` folder, like this:

![image](https://github.com/user-attachments/assets/f09a1790-5fd7-4f16-918b-7a985ddbc683)

**Reply by Alan0818zz** [reply]
Thanks,I slove it!!!
